app/src/main/java/fi/bitrite/android/ws/ui/UserFragment.java

```diff
@@ -211,14 +206,9 @@ void onFavoriteSelected(CompoundButton button, boolean isChecked) {
     }
 
     private void updateUserViewContent() {
-        MaybeNull<User> maybeUser = mUser.getValue();
-        mLayoutDetails.setVisibility(maybeUser.isNonNull() ? View.VISIBLE : View.GONE);
-
-        if (maybeUser.isNull()) {
-            return;
-        }
+        mLayoutDetails.setVisibility(View.VISIBLE);
```

Comment:
Make it `View.GONE` in `onCreateView`. Then we can remove that property from the layout file.